modules/ROOT/pages/clauses/load-csv.adoc

```diff
@@ -68,8 +70,7 @@ Added 4 nodes, Set 8 properties, Added 4 labels
 ====
 
 [NOTE]
-It's not possible to use `LOAD CSV` with file URLs in an Aura instance, only remote URLs are supported.
-For other ways of importing data into an Aura instance, see link:https://neo4j.com/docs/aura/auradb/importing/importing-data/[Aura -> Importing data].
+For ways of importing data into an Aura instance, see link:https://neo4j.com/docs/aura/auradb/importing/importing-data/[Aura -> Importing data].
```

Comment:
Might be worth updating the content at this link with something like

> For security reasons, you must host your .csv file on a publicly accessible HTTP or HTTPS server. Examples of such servers include AWS Signed URLs, GitHub, Google Drive, and Dropbox.

As the security side of S3 URIs for Aura isn't yet sorted at least the new bit shows they _could_ use it if they needed to now

Otherwise looks good 👍🏼